endpoints/telegram.py

```
@@ -18,52 +19,60 @@ def __init__(self, bot: telebot.TeleBot, users_service: UsersService, notificati
             sleep(int(self.config["giveTokensWhenStartAfterSeconds"]))
             
-            users_service.give_tokens(str(message.chat.id), int(self.config["giveTokensWhenStartAmount"]))
+            users_service.give_tokens(str(message.chat.id), int(self.config["giveTokensWhenStartAmount"]), ray_id)
                                       
-            self.notifications_service.send_message(str(message.chat.id), "Something going on")
+            self.notifications_service.send_message(str(message.chat.id), "Something going on", ray_id)
 
         @self.bot.message_handler(commands=['balance'])
         def handle_balance_message(message):
-            user_get_balance = users_service.get_balance(str(message.chat.id))
+            self.lastRayId += 1; ray_id = self.lastRayId
```

Ruff (0.8.2)

Multiple statements on one line (semicolon)

(E702)